tests/saw/spec/handshake/handshake_io_lowlevel.saw

```diff
@@ -107,9 +112,9 @@ let setup_connection = do {
    let corked_io = {{1 : [8]}};
    crucible_points_to (conn_corked_io pconn) (crucible_term corked_io); 
    
-   // TLS1.3 is temporarily removed from SAW testing, force the TLS1.2 version
-   let version = {{33 : [8]}};
-   crucible_points_to (crucible_field pconn "actual_protocol_version") (crucible_term version);
+   // this proof is only valid for the tls1.2 version of the handshake
+   let tls12_version = {{33 : [8]}};
+   crucible_points_to (crucible_field pconn "actual_protocol_version") (crucible_term tls12_version);
```

Comment:
Why are you modifying this? You don't actually change any behavior in this file or handshake.saw, so you might want to just remove them to keep the diff clean.